src/Module/Util/ZipHandler.php

```diff
@@ -57,27 +58,41 @@ public function isZipable(string $object_type): bool
 
     /**
      * takes array of full paths to medias
-     * zips them and sends them
+     * zips them, adds art and m3u, and sends them
      *
      * @param string $name name of the zip file to be created
      * @param array $media_files array of full paths to medias to zip create w/ call to get_media_files
      */
     public function zip(string $name, array $media_files): void
     {
+        $art     = (string) AmpConfig::get('album_art_preferred_filename');
+        $addart  = (bool) AmpConfig::get('art_zip_add');
```

Comment:
Two little issues related to the config. The usage of static calls `AmpConfig` is deprecated (not marked as deprecated yet though).

- Could you please add this strings as constants (with the same name; according to the pattern of the existing keys in the file) to the ConfigurationKeyEnum (located here `src/Config/ConfigurationKeyEnum.php`) and use the constants from there instead of string values?
You could use the following then:
```php
$addart = $this->configContainer->isFeatureEnabled(ConfigurationKeyEnum::ART_ZIP_ADD);
```

- Use the `get`-Method of the configContainer to retrieve values from the config
```php
$art = $this->configContainer->get(ConfigurationKeyEnum::ALBUM_ART_PREFERRED_FILENAME);
```

Reply:
so consequently i also change this one:
        $options = [
            'comment' => $this->configContainer->get('file_zip_comment'),
        ];
to
        $options = [
            'comment' => $this->configContainer->get(ConfigurationKeyEnum::FILE_ZIP_COMMENT),
        ];